aerogear/mobile-cli
# Update the CLI to accept xamarin client type

Changes proposed in this pull request
 - add xamarin client type

**Which issue this PR fixes (This will close that issue when PR gets merged)**
https://issues.jboss.org/browse/AEROGEAR-2404

##Verification Steps
make sure your openshift is running locally and mobile plugin is set up.
1. checkout this pr locally
2. `./mobile create client test xamarin {{NAME}} --namespace=myproject` - should show table with the client, Repeat this step few times with different names and ids
3. `./mobile get clients --namespace=myproject` should show created clients
4. `./mobile delete client {{ID}} --namespace=myproject` run get clients and ensure that it was deleted.
5. `./mobile get client {{ID}} --namespace=myproject`
6. Run unit and integration tests to ensure its passing
